tests/OrderableRowsThroughTest.yml

```diff
@@ -0,0 +1,40 @@
+Symbiote\GridFieldExtensions\Tests\Stub\ThroughDefiner:
+  DefinerOne:
+    ID: 1
+  DefinerTwo:
+    ID: 2
```

Comment:
Defining IDs in fixtures isn't a very good idea - you'd ideally let the DB driver assign them via auto increments. Maybe just define a simple varchar for the sake of fixturing instead?